open-policy-agent/service-account-mgmt/serviceaccount_automounttoken_prevention/constraint.yaml

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
@michaelkotelnikov @tommeramber 
What is the reason to excludeNamespaces in each policy instead of applying the exclusion in a general-config file?